### ISSUE TITLE
Implement the revised image permissions statement 

### ISSUE THREAD

**Issue by gissoo** [Princeton-CDH/geniza#775]
## Testing notes (qa -- round 2)

- [x] On the QA site, confirm focus styles are correct for the "Image Permissions Statement" buttons

---

## Testing notes (qa)

- [x] On the QA site, navigate to some documents with images, and confirm the image permissions statement link and list look correct.

---

[Link to mock-ups on Figma](https://www.figma.com/file/anZFStv7zBaVWnFxr95nKt/Work-in-Progress-%2F-WIP?node-id=1184%3A35954)

- [x] default view (collapsed) with downward phosphor icon chevron – on desktop and mobile
- [x] selected view (expanded) with upward phosphor icon chevron – on desktop and mobile
- [x] font styles and alignment for label: link style, below the transcription panel pagination, and bold shelfmark, taking up the full line when expanded – on desktop and mobile
- [x] spacing between the label and the transcription panel pagination and between each shelfmark when expanded – on desktop and mobile – marked in red on the mock-ups

**Comment by rlskoeser** [comment]
@blms please implement so it supports displaying both attribution and license; we don't currently have both, but it's feasible we will in future

**Comment by gissoo** [comment]
@rlskoeser I will email them this week about access and changes in our subscription, this is just unacceptable – I thought we agreed to display them this way because Marina mentioned the "shelfmark by shelfmark" and you said it made sense to you – is that not necessary anymore? – I can revise the designs and change the styling if that's the case – I'll create a chore

**Comment by rlskoeser** [comment]
sorry I didn't think of this before, @gissoo!  It seemed reasonable to me to be explicit about rights and permissions when there are multiple shelfmarks, since the attribution and license may differ (probably by instutition/source).  It's also a bit more logic for us to figure out how to aggregate them, since the order may not align with the source.

I'm not sure how important it is that the logos be tied to specific shelfmarks — I think it's more important to display them once, and would prefer to display them only once.

**Comment by gissoo** [comment]
@blms thank you!! The sizes and styles look correct, but some have the image/logo and some like the below don't. Are there cases where there is more than one shelfmark and one logo/right image only? I can't find one to test. If there are no cases like that or you don't want me to test those please feel free to close this PR. 

<img width="947" alt="Screen Shot 2022-04-22 at 6 31 07 PM" src="https://user-images.githubusercontent.com/44615389/164807168-abd1ca7c-b6ee-43f4-95ee-562050e11e65.png">

**Comment by gissoo** [comment]
@blms just caught this, could you remove the focus indicator on click? it's quite distracting – if you can't replicate this please ignore my comment. 

<img width="565" alt="Screen Shot 2022-04-22 at 6 44 08 PM" src="https://user-images.githubusercontent.com/44615389/164814093-54fe96f2-bfe0-4d32-8fc4-7b8548ffcd67.png">

<img width="1103" alt="Screen Shot 2022-04-22 at 6 44 40 PM" src="https://user-images.githubusercontent.com/44615389/164814138-1dec416a-45ec-41b8-9d23-49a690e2e533.png">

**Comment by blms** [comment]
Thanks, I'll work on the focus indicator!

> @blms thank you!! The sizes and styles look correct, but some have the image/logo and some like the below don't. Are there cases where there is more than one shelfmark and one logo/right image only? I can't find one to test. If there are no cases like that or you don't want me to test those please feel free to close this PR.

I don't think there are any cases that have both. Also I think it's worth documenting the three different types of thing we might see here:
- Attribution ("Provided by Cambridge University Library" etc.—implemented)
- License/rights statement (images e.g. "No Known Copyright"—implemented)
- Logos (images—not yet implemented)

@rlskoeser Does that sound right, and is it indeed true that there aren't any fragments that have both an attribution and a license/rights statement? I didn't find any using this django shell command:

```py
frags = Fragment.objects.filter(manifest__isnull=False)
for f in frags:
    if f.manifest.license and f.attribution:
        print(f.pk)
```

**Comment by blms** [comment]
> I was planning to handle that in a separate issue as a follow up to this one — does it have a bearing on the design/style questions?

@gissoo can answer better, but I imagine it does insofar as the designs need to account for some fragments having both attribution and rights statement. Should these rights statement images be inline with each fragment, or should they be collected at the bottom like the proposed solution for logos? I imagine inline is preferable for that, but not sure.

**Comment by rlskoeser** [comment]
> Should these rights statement images be inline with each fragment, or should they be collected at the bottom like the proposed solution for logos? I imagine inline is preferable for that, but not sure.

Forgot to respond to this question — in the current design I think rights statements need to be inline so people can tell which rights apply to which shelfmark. Gissoo may revisit this in the new designs, but let's try to keep this simple for now.

**Comment by gissoo** [comment]
@rlskoeser @blms I think if you could give me some specific requirements for these I can do them quickly or we can turn them into new issues and break this down.

**Comment by gissoo** [comment]
@rlskoeser @blms I think if you could give me some specific requirements for these here or on Slack I can do them quickly and give them to you via some design chores or issues, or we can turn them into separate implementation issues and break this down.

**Comment by rlskoeser** [comment]
@gissoo I can talk to you about it — LMK what would be easiest for you (can we design for the new version though instead of the interim solution?)

**Comment by gissoo** [comment]
@rlskoeser @blms your comments make sense. Thank you! I'll make an issue to address this along with the transcription panel issues. Everything else looks fine.
